inc/document_item.class.php

```diff
@@ -85,14 +85,26 @@ function canCreateItem() {
 
    function prepareInputForAdd($input) {
 
+      if (empty($input['itemtype']) || trim($input['itemtype']) == '') {
+         Toolbox::logError('Item type is mandatory');
+         return false;
+      }
+
       if ((empty($input['items_id']) || ($input['items_id'] == 0))
           && ($input['itemtype'] != 'Entity')) {
+         Toolbox::logError('Item ID is mandatory');
+         return false;
+      }
+
+      if (empty($input['documents_id']) || ($input['documents_id'] == 0)) {
```

Comment:
`0` will be considered as empty, no need for the `($input['items_id'] == 0)` test

Reply:
Indeed, I've dropped useless checks